src/rpc/client_stream.rs

```diff
@@ -69,14 +69,11 @@ impl WebRTCClientStream {
             }
         };
 
-        self.base_stream.close_with_recv_error(&mut err.as_ref());
-        match err {
-            None => Ok(()),
-            Some(err) => {
-                log::error!("Error processing trailers: {err}");
-                Err(err)
-            }
+        if let Some(e) = &err {
+            log::error!("received gRPC error: {e}");
```

Comment:
I went with `error` level logging here just to increase visibility and because it is definitely an error. However, this may be overkill since we'll still be getting an error from `call`? Happy to update to `debug` level if people think that's preferred.

Reply:
I say debug!

Reply:
+1 for debug

Reply:
debug it is!